crates/swc_ecma_minifier/tests/fixture/issues/6957/number-tostring/output.js

```diff
@@ -0,0 +1,291 @@
+assertEquals("-1e-7", "-1e-7");
+assertEquals("-1.2e-7", "-1.2e-7");
+assertEquals("-1.23e-7", "-1.23e-7");
+assertEquals("-1e-8", "-1e-8");
+assertEquals("-1.2e-8", "-1.2e-8");
+assertEquals("-1.23e-8", "-1.23e-8");
+assertEquals("NaN", NaN.toString(16));
+assertEquals("Infinity", (1 / 0).toString(16));
+assertEquals("-Infinity", (-1 / 0).toString(16));
+assertEquals("0", "0");
+assertEquals("9", "9");
+assertEquals("5a", "5a");
+assertEquals("5a.1eb851eb852", 90.12.toString(16));
+assertEquals("0.1999999999999a", 0.1.toString(16));
+assertEquals("0.028f5c28f5c28f6", 0.01.toString(16));
+assertEquals("0.032617c1bda511a", 0.0123.toString(16));
+assertEquals("605f9f6dd18bc8000", "605f9f6dd18bc8000");
+assertEquals("3c3bc3a4a2f75c0000", "3c3bc3a4a2f75c0000");
+assertEquals("25a55a46e5da9a00000", "25a55a46e5da9a00000");
+assertEquals("0.0000a7c5ac471b4788", 0.00001.toString(16));
+assertEquals("0.000010c6f7a0b5ed8d", 0.000001.toString(16));
+assertEquals("0.000001ad7f29abcaf48", 0.0000001.toString(16));
+assertEquals("0.000002036565348d256", 0.00000012.toString(16));
+assertEquals("0.0000021047ee22aa466", 0.000000123.toString(16));
+assertEquals("0.0000002af31dc4611874", 0.00000001.toString(16));
+assertEquals("0.000000338a23b87483be", 0.000000012.toString(16));
+assertEquals("0.00000034d3fe36aaa0a2", 0.0000000123.toString(16));
+assertEquals("0", "0");
+assertEquals("-9", "-9");
+assertEquals("-5a", "-5a");
+assertEquals("-5a.1eb851eb852", (-90.12).toString(16));
+assertEquals("-0.1999999999999a", (-0.1).toString(16));
+assertEquals("-0.028f5c28f5c28f6", (-0.01).toString(16));
+assertEquals("-0.032617c1bda511a", (-0.0123).toString(16));
+assertEquals("-605f9f6dd18bc8000", "-605f9f6dd18bc8000");
+assertEquals("-3c3bc3a4a2f75c0000", "-3c3bc3a4a2f75c0000");
+assertEquals("-25a55a46e5da9a00000", "-25a55a46e5da9a00000");
+assertEquals("-0.0000a7c5ac471b4788", (-0.00001).toString(16));
+assertEquals("-0.000010c6f7a0b5ed8d", (-0.000001).toString(16));
+assertEquals("-0.000001ad7f29abcaf48", (-0.0000001).toString(16));
+assertEquals("-0.000002036565348d256", (-0.00000012).toString(16));
+assertEquals("-0.0000021047ee22aa466", (-0.000000123).toString(16));
+assertEquals("-0.0000002af31dc4611874", (-0.00000001).toString(16));
+assertEquals("-0.000000338a23b87483be", (-0.000000012).toString(16));
+assertEquals("-0.00000034d3fe36aaa0a2", (-0.0000000123).toString(16));
+assertEquals("4294967296", "4294967296");
+assertEquals("ffffffff", "ffffffff");
+assertEquals("11111111111111111111111111111111", "11111111111111111111111111111111");
+assertEquals("5yc1z", "5yc1z");
+assertEquals("0", "0");
+assertEquals("0", "0");
+assertEquals("0", "0");
+assertEquals("0", "0");
+assertEquals("0", "0");
+assertEquals("100000000000000000000000000000000", "100000000000000000000000000000000");
+assertEquals("100000000000000000000000000000001", "100000000000000000000000000000001");
+assertEquals("100000000000080", "100000000000080");
+assertEquals("1000000000000100", "1");
```

Comment:
I think this will fail

Reply:
```JavaScript 
assertEquals("1000000000000100", (-(-'0x1000000000000081')).toString(16));
```
Yes. It seems that somewhere the calculation result of `(-(-'0x1000000000000081'))` is wrong. 

[Check it](https://play.swc.rs/?version=1.3.35&code=H4sIAAAAAAAAA0ssLk4tKnEtLE3MKdZQMjRAAkCOko6Chq6GrrpBBYqMgYWhuqamXkl%2BcElRZl66hqGZpqY1AAUpUdlKAAAA&config=H4sIAAAAAAAAA22TQZLjIAxF78K6F11ZzKIP0Ls%2BA0VAOGQwciGRjiuVu49wbIiT2VnPX0J8iZs6k1VfNzWZTJDrF82JzVV9KZ4nIJvDxOpDMQnyJhLcJTB5ABYJ0OHzcJDfEZFgFXyoMaTg51rM4jhlIKrfklRGSExNZ3LGXwk5F4mOiBFMeou1IR0SwwC5p1qM0UwE%2BmJyy6inmRwI0w4VBqenjNMTTS5wwCTFNubAOG3RQQMhg%2BVwgS6RGiJJJI21Thbo4FiGoRr4UMLFxGK41YLrYoOc2PJOGIi1L4le0K7TB3pcclUFrzNwyWnTnDGknQ9%2FAaTPaIiSGaFnLtzLHPZK%2Fx9VSF6GyHOjMuHeVYJBLqdD8O2GtWvIHLpbGVyxUG9oe%2FEV7i5EwYEG78Xtlky%2Fge2pF6vLiL6F4prxfQaPUG%2F7tKd1Bd7gt3TFz%2BNY%2BWj49MpoHo8Y30qMwCd0b1iMYHyFWbb3Or3SkhyIzeCefhSqYVtTGT6jjvW1bT7LQkiOHiIe2%2FrKoxxNGtpa3itAVyq4LeYtr%2FWPuvfXua0T%2FazCpdA%2FOW4Z9xIEAAA%3D)

I can't locate the wrong code right now.
Maybe it should be fixed in another Pull Request.

Reply:
I see, then let's postpone it